os/Kconfig

```diff
@@ -241,6 +241,15 @@ config AUTOGEN_MEMORY_LDSCRIPT
 
 endif # BUILD_PROTECTED
 
+config MPU_STACKGUARD
```

Comment:
This implementation is ARM-specific. Please move this in Kconfig of ARM.

Reply:
done